.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Thank you for contributing to The Foreman, please read the following guide
```

Comment:
Is it worth putting this as a comment using `<!---` and `--->` so people don't have to delete it when raising a PR

Reply:
Do you mean completely whole file contents? I can do that.

Reply:
Yes 👍

Reply:
Made the change, I left two blank line at the very bottom so ppl can start typing right away.